smithy-aws-protocol-tests/model/restJson1/malformedRequests/malformed-accept.smithy

```diff
@@ -0,0 +1,133 @@
+$version: "1.0"
+
+namespace aws.protocoltests.restjson
+
+use aws.protocols#restJson1
+use smithy.test#httpMalformedRequestTests
+use aws.protocoltests.shared#GreetingStruct
+use aws.protocoltests.shared#JpegBlob
+
+apply MalformedAcceptWithoutBody @httpMalformedRequestTests([
+    {
+        id: "RestJsonWithoutBodyExpectsEmptyAccept",
+        documentation: """
+        When there is no modeled output, accept must not be set.""",
```

Comment:
Do we want to be this strict? If a client that handles a JSON protocol set `Accept: application/json` on all requests without payload or event stream binding on the response I don't think I'd be too upset with that behavior.

Reply:
I guess I can go either way.